app/experimenter/nimbus-ui/src/components/FormAudience/index.tsx

```diff
@@ -184,33 +290,43 @@ export const FormAudience = ({
             </Form.Label>
             <InputGroup className="mb-3">
               <Form.Control
-                placeholder="28"
-                aria-describedby="experimentDuration-unit"
-                defaultValue={proposedDuration || 28}
+                {...formControlCommon("proposedDuration")}
+                type="number"
+                aria-describedby="proposedDuration-unit"
               />
               <InputGroup.Append>
-                <InputGroup.Text id="experimentDuration-unit">
+                <InputGroup.Text id="proposedDuration-unit">
                   days
                 </InputGroup.Text>
               </InputGroup.Append>
+              <FormErrors name="proposedDuration" />
             </InputGroup>
           </Form.Group>
         </Form.Row>
       </Form.Group>
 
       <div className="d-flex flex-row-reverse bd-highlight">
         <div className="p-2">
-          <button data-testid="next-button" className="btn btn-secondary">
+          <button
+            onClick={handleNext}
+            className="btn btn-secondary"
+            disabled={isLoading}
+            data-testid="next-button"
+            data-sb-kind="pages/RequestReview"
+          >
             Next
           </button>
```

Comment:
I don't think we do this on other forms - is there an example elsewhere of this?

Reply:
We don't do this on other forms only because their Next buttons would take you to another form; this Next button, on the other hand, takes you to the Review page, which should or should not be accessible until the experiment is ready to be submitted for review